admin/common_auth/models.py

```diff
@@ -47,6 +47,10 @@ class MyUser(AbstractBaseUser, PermissionsMixin):
     date_joined = models.DateTimeField(default=datetime.now)
     confirmed = models.BooleanField(default=False)
     osf_id = models.CharField(default=False, max_length=10, blank=True)
+    desk_email = models.EmailField(verbose_name='desk email', max_length=255,
+                                   default='')
+    desk_password = models.CharField(verbose_name='desk password',
+                                     max_length=128, default='')  # TODO: encrypt
```

Comment:
After discussion, it may be best to just the user-specific access keys provided by desk, and keep the application-specific oauth keys in a `/settings/local.py` file.